02.calendar/cal.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby  
+require "date"
+require "optparse"
+
+default_date = Date.today
+
+opt = OptionParser.new
+option = {}
+opt.on("-y year") {|year| option[:y] = year}
+opt.on("-m month") {|month| option[:m] = month}
+opt.parse!(ARGV)
+
+year = option[:y] ? option[:y].to_i : default_date.year
+month = option[:m] ? option[:m].to_i : default_date.month
+
+start_date = Date.new(year, month)
```

Comment:
```suggestion
start_date = Date.new(year, month, 1)
```

明示的に1を指定した方がわかりやすいと思います

Reply:
引数に1を指定しました。

```ruby
start_date = Date.new(year, month, 1)
```